Mathlib/MeasureTheory/Function/LpSeminorm.lean

```diff
@@ -1601,29 +1628,28 @@ variable [MeasurableSpace E] [OpensMeasurableSpace E] {R : ℝ≥0}
 
 theorem ae_bdd_liminf_atTop_rpow_of_snorm_bdd {p : ℝ≥0∞} {f : ℕ → α → E}
     (hfmeas : ∀ n, Measurable (f n)) (hbdd : ∀ n, snorm (f n) p μ ≤ R) :
-    ∀ᵐ x ∂μ, liminf (fun n => (‖f n x‖₊ ^ p.toReal : ℝ≥0∞)) atTop < ∞ := by
-  by_cases hp0 : p.to_real = 0
+    ∀ᵐ x ∂μ, liminf (fun n => ((‖f n x‖₊ : ℝ≥0∞) ^ p.toReal : ℝ≥0∞)) atTop < ∞ := by
+  by_cases hp0 : p.toReal = 0
   · simp only [hp0, ENNReal.rpow_zero]
     refine' eventually_of_forall fun x => _
     rw [liminf_const (1 : ℝ≥0∞)]
-    exacts[ENNReal.one_lt_top, at_top_ne_bot]
-  have hp : p ≠ 0 := fun h => by simpa [h] using hp0
-  have hp' : p ≠ ∞ := fun h => by simpa [h] using hp0
+    exact ENNReal.one_lt_top
+  have hp : p ≠ 0 := fun h => by simp [h] at hp0
+  have hp' : p ≠ ∞ := fun h => by simp [h] at hp0
```

Comment:
I think this could do with a porting note about `simpa` not working, unless you know why and it's a stupid reason!

Reply:
Because the linter warns to change from `simpa` to `simp` for efficiency.